gmso/formats/lammpsdata.py

```diff
@@ -654,13 +719,437 @@ def _get_ff_information(filename, unit_style, topology):
+                atom_type.name,
+            )
+        )
+
+
+def _write_pairtypes(out_file, top, base_unyts, cfactorsDict):
+    """Write out pair interaction to LAMMPS file."""
+    # TODO: Modified cross-interactions
+    # TODO: Utilize unit styles and nonbonded equations properly
+    # Pair coefficients
+    test_atmtype = top.sites[0].atom_type
+    out_file.write(
+        f"\nPair Coeffs # lj\n"
+    )  # TODO: This should be pulled from the test_atmtype
+    # TODO: use unit style specified for writer
+    param_labels = map(
+        lambda x: f"{x} ({test_atmtype.parameters[x].units})",
+        ("epsilon", "sigma"),
+    )
+    out_file.write("#\t" + "\t".join(param_labels) + "\n")
+    sorted_atomtypes = sorted(
+        top.atom_types(filter_by=pfilter), key=lambda x: x.name
+    )
+    for idx, param in enumerate(sorted_atomtypes):
+        # TODO: grab expression from top
+        out_file.write(
+            "{}\t{:7.5f}\t\t{:7.5f}\t\t# {}\n".format(
+                idx + 1,
+                _parameter_converted_to_float(
+                    param.parameters["epsilon"], base_unyts, cfactorsDict
+                ),
+                _parameter_converted_to_float(
+                    param.parameters["sigma"], base_unyts, cfactorsDict
+                ),
+                param.name,
+            )
+        )
+
+
+def _write_bondtypes(out_file, top, base_unyts, cfactorsDict):
+    """Write out bonds to LAMMPS file."""
+    # TODO: Use any accepted lammps parameters
+    test_bontype = top.bonds[0].bond_type
+    out_file.write(f"\nBond Coeffs #{test_bontype.name}\n")
+    param_labels = map(
+        lambda x: f"{x} ({test_bontype.parameters[x].units})",
+        test_bontype.parameters,
+    )
+    out_file.write("#\t" + "\t".join(param_labels) + "\n")
+    bond_types = list(top.bond_types(filter_by=pfilter))
+    bond_types.sort(key=lambda x: sorted(x.member_types))
+    for idx, bond_type in enumerate(bond_types):
+        member_types = sorted(
+            [bond_type.member_types[0], bond_type.member_types[1]]
+        )
+        out_file.write(
+            "{}\t{:7.5f}\t{:7.5f}\t\t# {}\t{}\n".format(
+                idx + 1,
+                _parameter_converted_to_float(
+                    bond_type.parameters["k"], base_unyts, cfactorsDict
+                ),
+                _parameter_converted_to_float(
+                    bond_type.parameters["r_eq"], base_unyts, cfactorsDict
+                ),
+                *member_types,
+            )
+        )
+
+
+def _write_angletypes(out_file, top, base_unyts, cfactorsDict):
+    """Write out angles to LAMMPS file."""
+    # TODO: Use any accepted lammps parameters
+    test_angtype = top.angles[0].angle_type
+    test_angtype.parameters["theta_eq"].convert_to_units("degree")
+    out_file.write(f"\nAngle Coeffs #{test_angtype.name}\n")
+    param_labels = map(
+        lambda x: f"{x} ({test_angtype.parameters[x].units})",
+        test_angtype.parameters,
+    )
```

Comment:
as above

Reply:
Fixed in 01471a7c1ef4b586eaa52fe327670fcee2feff5b